### DISCUSSION TITLE
https://widcard.win/posts/notes/details-about-shader-of-bezier-curve#comments

### DISCUSSION THREAD

**Discussion by giscus** [widcardw/giscus-discussion#17] [Announcements]
# https://widcard.win/posts/notes/details-about-shader-of-bezier-curve#comments

Opinionated Vite Starter Template

https://widcard.win/posts/notes/details-about-shader-of-bezier-curve

**Comment by hannahwang37** [comment]
:) 请问着色流程那张图片是博主原创的吗，很好看。

**Reply by widcardw** [reply]
是阅读 3b1b/manim 的 shaders 部分源码，然后把流程精简了一下，绘制出来的，算是一个总结的图。
